### PR TITLE
working on branch graph

### PR DESCRIPTION
**Pull request by rbral**

### CODE REVIEW

File: src/main/java/bral/projectile/ProjectileGraph.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            repaint();
+        }
+
+    }
+
+    public void setProjectile(Projectile projectile)
+    {
+        this.projectile = projectile;
+
+        // tells the OS to call painComponent again:
```
Comment (andrewoid) [inline src/main/java/bral/projectile/ProjectileGraph.java]: `paintComponent`

**Comment by andrewoid** [comment]
Add a screenshot.

File: src/main/java/bral/projectile/ProjectileGraph.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                    (int) newX, (int) -newY);
+            oldX = newX;
+            oldY = newY;
+
+            if (Math.abs(i - projectile.getApexTime()) < 0.05)
+            {
+                g.setColor(Color.BLUE);
+                g.fillOval((int) newX - 5, (int) -newY - 5, 10, 10);
+                g.setColor(Color.DARK_GRAY);
+            }
```
Comment (andrewoid) [inline src/main/java/bral/projectile/ProjectileGraph.java]: This should be done a better way.

**Comment by rbral** [comment]
Screenshot 2024-03-06 165625

**Review by andrewoid — APPROVED** [review]

**Pull request merged**